sse/src/test/scala/akka/stream/alpakka/sse/EventSourceSpec.scala

```diff
@@ -0,0 +1,154 @@
+/*
+ * Copyright (C) 2016 Lightbend Inc. <http://www.lightbend.com>
+ */
+package akka.stream.alpakka.sse
```

Comment:
Should this be in `scaladsl` to follow the convention of other things?

Missing javadsl?